templates/rbac.yaml

```diff
@@ -6,14 +6,17 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   namespace: {{ .Release.Namespace | quote }} 
-  name: environments
```

Comment:
Do we not use this service account t anywhere? I thought it was mounted for the default workspace provider.

Reply:
@kylecarbs i believe just the `coder` service account is mounted to the built-in provider.

i just tested things, and it looks like every workspace is provisioned it's own respective SA, and this `environments` SA is not mounted as the service account. we may need to make some back-end changes to have workspaces default to using this `environments` SA.

thoughts?